pkg/virt-controller/watch/replicaset_test.go

```diff
@@ -174,6 +174,42 @@ var _ = Describe("Replicaset", func() {
 			// TODO test for missing 5
 		})
 
+		It("should already create replacement VMIs once it discovers that a VMI is marked for deletion", func() {
+			rs, vmi := DefaultReplicaSet(10)
```

Comment:
A minor point, could you use named variables for the total count of 10 here and the expected difference of 7 below? It took a moment to figure out where the magic number 7 came from.